### ISSUE TITLE
Standard deviation

### ISSUE THREAD

**Issue by jrcblue** [commaai/openpilot#2143]
I am using Openpilot to do some simulation and I was wondering what is the meaning of the standard deviation in the output of the deep neural network. For instance, if a waypoint's std is 4, how to interpret it or use the information. Thanks!

**Comment by HaraldSchafer** [comment]
The distribution of the error of the output of a certain values compared to the ground truth has a mean of 0 and the standard deviation reported. So it is essentially a prediction of the size of the error expected.

**Comment by jrcblue** [comment]
Thanks! One more question about the std: is the std prediction is trained by the mean negative log-likelihood loss function?

**Comment by HaraldSchafer** [comment]
Yes it is!
